app/src/test/java/pjwstk/receipemate/app/viewrepository/category/CategoryViewRepositoryUnitTest.java

```diff
@@ -0,0 +1,86 @@
+package pjwstk.receipemate.app.viewrepository.category;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageImpl;
+import org.springframework.data.domain.Pageable;
+import pjwstk.receipemate.app.entity.Category;
+import pjwstk.receipemate.app.exception.NotFoundException;
+import pjwstk.receipemate.app.repository.category.CategoryRepository;
+import pjwstk.receipemate.app.view.category.CategoryView;
+import pjwstk.receipemate.app.view.page.PageView;
+import pjwstk.receipemate.app.viewfactory.category.CategoryViewFactory;
+import pjwstk.receipemate.app.viewfactory.page.PageViewFactory;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class CategoryViewRepositoryUnitTest {
+
+    @Mock
+    CategoryRepository categoryRepositoryMock;
+
+    @Mock
+    CategoryViewFactory categoryViewFactoryMock;
+
+    @Mock
+    PageViewFactory pageViewFactoryMock;
+
+    @InjectMocks
+    CategoryViewRepository categoryViewRepository;
+
+
+    @Test
+    public void getList_shouldReturnPageViewOfCategories() {
+        //given
+        Pageable pageable = Pageable.unpaged();
+        List<Category> categories = new ArrayList<>();
```

Comment:
Zastanawiam się czy nie lepiej będzie zamiast tworzenia listy zostosować po prostu List.of(category1, category2). Co uważasz?

Reply:
done